.cursor/rules/accessibility-docs.mdc

```diff
@@ -0,0 +1,244 @@
+---
+description: Revision, creation and modification of accessibility documentation files
+globs: 
+alwaysApply: false
+---
+# Accessibility Documentation Rules
+
+## Purpose and Scope
+
+The accessibility documentation should:
+- Focus on how OUR components implement and handle accessibility
+- Document built-in/automatic accessibility features
+- Explain component-specific configuration options
+- Provide practical usage examples specific to our components
+
+The documentation should NOT:
+- Explain general ARIA concepts or accessibility guidelines
+- Provide general education about what roles or attributes mean
+- Include generic accessibility best practices not specific to our components
+
+## Before Starting
+
+1. Ask for any specific requirements or notes for the component
+2. Review existing JIRA task notes for additional context
+3. Check if component has special accessibility considerations
+
+## File Structure and Location
+
+- Accessibility documentation files should be named `03-accessibility.mdx`
+- Files should be placed in `docs/src/documentation/03-components/<component-name>/`
+- Component source code is in `packages/orbit-components/src/<component-name>/`
+- Component README files are located next to component source code
+
+## File Format and Structure
+
+### Required Frontmatter
+
+```mdx
+---
+title: Accessibility
+redirect_from:
+  - /components/<component-name>/accessibility/
+---
+```
+
+### Document Structure
+
+#### Required Section Names
+
+Documentation must use these exact section names and structure:
+
+1. Main heading and introduction:
+```mdx
+## Accessibility
+
+The [ComponentName] component has been designed with accessibility in mind.
+```
+
+2. For multiple components:
+```mdx
+## Accessibility
+
+### ComponentOne
+The ComponentOne component has been designed with accessibility in mind.
+[ComponentOne documentation...]
+
+### ComponentTwo
+The ComponentTwo component has been designed with accessibility in mind.
+[ComponentTwo documentation...]
+```
+
+3. Props documentation:
+```mdx
+#### Accessibility Props
+
+| Name | Type | Description |
+| :--- | :--- | :---------- |
+```
+
+4. For components with multiple elements:
+```mdx
+#### Accessibility Props
+
+**Input field props:**
+
+| Name | Type | Description |
+| :--- | :--- | :---------- |
+| ariaLabel | string | Specifies aria-label for the input field |
+
+**Button props:**
+
+| Name | Type | Description |
+| :--- | :--- | :---------- |
+| titleDecrement | string | Specifies aria-label for the decrement button |
+```
+
+5. Best practices (if applicable):
+```mdx
+#### Best practices
+
+- Guideline 1
+- Guideline 2
+```
+
+6. Code examples and screen reader behavior:
+```mdx
+#### Code examples
```

Comment:
As defended in my PR, I think it makes more sense to have examples next to the place where the aspects are described. And best practices as a summary at the end of the file. I'd leave it for democracy to chose